crates/typed-store/src/rocks/mod.rs

```diff
@@ -298,8 +339,17 @@ where
 
     #[instrument(level = "trace", skip_all, err)]
     fn get(&self, key: &K) -> Result<Option<V>, TypedStoreError> {
+        let _timer = self
```

Comment:
Again the timer here is unnecessary.  We should be able to get that today

Reply:
i think i mentioned before, but this tracks metrics by db/column family which is useful and not possible with spans